### PR TITLE
Disable mayaonprem feature and disable subscription

### PR DESCRIPTION
**Pull request by inyee786**
- Disable mayaonprem feature
 - Disable subscription

Signed-off-by: inyee786 <intakhab.ali@mayadata.io>

### CODE REVIEW

File: 1.0.0/templates/maya-config.yaml [hunk 1]
```diff
@@ -50,6 +50,7 @@ data:
     maya.docs.url={{ .Values.server.docsUrl }}
     feature.mdape2e.disable=true
     feature.tour.disable=true
+    feature.mayaonpremregistration.disable=true
```
Comment (Moteesh) [inline 1.0.0/templates/maya-config.yaml]: Should the value be  using the values.yaml variable 'featureSubscriptionDisable'

Comment (inyee786) [reply 1.0.0/templates/maya-config.yaml]: no, as is is mayaonlline specific feature, we should not put it in value.yaml

Comment (inyee786) [reply 1.0.0/templates/maya-config.yaml]: i didn't get you, 
are you saying to get the above value `feature.mayaonpremregistration.disable`  from value.yaml?

Comment (Moteesh) [reply 1.0.0/templates/maya-config.yaml]: Can you please mention the purpose of the above variable and that of changing featureSubscriptionDisable to false ? I did not understand that part.

**Pull request closed without merging**